### ISSUE TITLE
error when run demo.py

### ISSUE THREAD

**Issue by Hrhythm** [xingyizhou/CenterNet#36]
when I run the demo.py, I met this error:
RuntimeError: cublas runtime error : the GPU program failed to execute at /pytorch/aten/src/THC/THCBlas.cu:416

How can I solve it, thank you!

**Comment by xingyizhou** [comment]
I have no idea unless more information is provided. You can check this https://github.com/pytorch/pytorch/issues/7548 .

**Comment by Aaronwd** [comment]
I met the same problem in the environment for pytorch 1.0 and cuda 10.1. I have no idea to deal with it. Thank you for your any support or suggestion.

**Comment by numarf** [comment]
from external.nms import soft_nms
ModuleNotFoundError: No module named 'external.nms'

**Comment by wangyuxin87** [comment]
> from external.nms import soft_nms
> ModuleNotFoundError: No module named 'external.nms'
Have you solved it? I have the same problem.

**Comment by numarf** [comment]
Yes. I made my own nms.py file, if you want that i can post the code here for you.
Thank You

**Comment by wangyuxin87** [comment]
@numarf Thanks a lot！You can send to my email   wangyx5858@163.com

**Comment by leiyaohui** [comment]
ModuleNotFoundError: No module named 'external.nms'
Hello, I have the same problem. How can you solve it?
I look forward to your reply. Thank you.

**Comment by numarf** [comment]
yep
please mail me at umarfarooq536@gmail.com for more details

**Comment by liushuai839** [comment]
> when I run the demo.py, I met this error:
> RuntimeError: cublas runtime error : the GPU program failed to execute at /pytorch/aten/src/THC/THCBlas.cu:416
> 
> How can I solve it, thank you!
Hi，Guys! I met the same problem in pytorch 0.4.1 and cuda9.0. Could you give me some advice? Thank you

**Comment by yuan243212790** [comment]
@liushuai839 i hava the same question. you kown how to solve this probem?
